### ISSUE TITLE
RGCN entity classification, incorrect loss function?

### ISSUE THREAD

**Issue by pbloem** [dmlc/dgl#1204]
## 🐛 Bug

In the RGCN [entity classification example](https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn/entity_classify.py), the model has a softmax activation, but the loss function, [F.cross_entropy](https://pytorch.org/docs/stable/nn.functional.html#cross-entropy), expects raw logits.

Unless I'm mistaken, either the last layer should have no activation, or the loss should be changed to F.nll_loss.

## To Reproduce

Running the example. Setting the activation to None has no effect on the score for AIFB. I can't easily test larger data like AM atm. If I add a double softmax to my own implementation of RGCN the performance on AM completely collapses (but then, I'm still debugging it).

**Comment by pbloem** [comment]
I've now debugged my own implementation, and having a double softmax doesn't seem to affect performance significantly (which explains why your numbers are still good). Still, it's technically more correct to ditch the activation (or use nll_loss).

**Comment by jermainewang** [comment]
Thanks for the report. You are correct. Will fix it.

-Minjie
